src/pages/Dashboard/components/ProfileAvatar/NftRow.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@chakra-ui/react'
+import type { GridChildComponentProps } from 'react-window'
+import type { NftItemWithCollection } from 'state/apis/nft/types'
+
+import { AvatarRadio } from './AvatarRadio'
+
+type NftRowProps = {
+  filteredNfts: NftItemWithCollection[]
+  columnCount: number
+  getRadioProps: any
```

Comment:
Can we type this?

Reply:
This is the type but its not exported `RadioPropGetter`